mico-admin/src/app/app-detail-status/app-detail-status.component.ts

```diff
@@ -21,17 +21,15 @@ export class AppDetailStatusComponent implements OnChanges, OnDestroy {
     applicationStatus;
 
     ngOnChanges() {
-        if (this.version != null) {
-
-            if (this.subApplicationStatus != null) {
-                this.unsubscribe(this.subApplicationStatus);
-            }
-
-            this.subApplicationStatus = this.apiService.getApplicationStatus(this.shortName, this.version)
-                .subscribe(val => {
-                    this.applicationStatus = val;
-                });
+        if (this.version == null || this.shortName == null) {
+            return;
         }
+
+        this.unsubscribe(this.subApplicationStatus);
+        this.subApplicationStatus = this.apiService.getApplicationStatus(this.shortName, this.version)
+            .subscribe(val => {
+                this.applicationStatus = JSON.parse(JSON.stringify(val));
```

Comment:
it should work, however someone might wants to use the view later to change things (e.g. deployment information like parameters). Hence, I would stick to the deep copy. Since the tab is loaded in background, the performance should not be a problem.